src/components/Footer.tsx

```diff
@@ -1,77 +1,166 @@
 import * as React from "react";
 
-const Footer = () => {
+export interface FooterProps {
+  _site?: any;
+  logo?: string;
+  paragraph?: string;
+}
+
+const currentTime = new Date()
+const year = currentTime.getFullYear()
+
+const navigation = {
+  company: [
+    { name: 'About', href: '#' },
+    { name: 'Blog', href: '#' },
+    { name: 'Jobs', href: '#' },
+    { name: 'Press', href: '#' },
+    { name: 'Partners', href: '#' },
+  ],
+  legal: [
+    { name: 'Claim', href: '#' },
+    { name: 'Privacy', href: '#' },
+    { name: 'Terms', href: '#' },
+  ],
+  social: [
+    {
+      name: 'Facebook',
+      href: '#',
+      icon: (props) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
```

Comment:
Would be nice to name this as a const and use it here which would make this easier to read.